File: java_backup/my java/romit/pattern1.java
import java.io.*;
import java.lang.*;
class pattern1
{
public static void main(String args[])throws IOException
{
BufferedReader Input=new BufferedReader(new InputStreamReader(System.in));
System.out.println("ENTER THE NUMBER");
int n=Integer.parseInt(Input.readLine());
int i,j,m;
for(i=1;i<=n;i++)
{
if(i%2==0)
{
m=0;
}
else
{
m=1;
}
for(j=1;j<i;j++)
{
System.out.print(m);
System.out.println(' ');
if(m==0)
{
m=1;
}
if(m==1)
{
m=0;
}
System.out.println(' ');
}
}
}
}

File: 4rth_Sem_c++_backup/Amlyan/c++/pattern1.cpp
#include<iostream>
#include<cmath>
using namespace std;
class pattern1
{
	int n;
	public:
		void input()
		{
			cout<<"Enter a number";
			cin>>n;
		}
		void pattern()
		{
			for(int i=1;i<=2*n+1;i++)
			{
				char ch='-';
				if(i%2==0)
				{
					ch='=';
				}
				cout<<"\n";
				if(i==1||i==2*n+1)
				{
					cout<<"+";
					for(int k=1;k<=2*n;k++)
					{
						cout<<"-";
					}
					cout<<"+";
				}
				else{
				
				    if(i==n+1)
				    {
				    	cout<<"|<";
				    	for(int j=1;j<2*n-1;j++)
					    {
				    		cout<<ch;
				    	}
				    	cout<<">|";
				   }
				 else 
				    {
				    	int e=n+2-i;
				    	char dh='/';
				    	char lh='\\';
				    	if(i>n+1)
				    	{
				    		e=i-n;
				    		dh='\\';
				     		lh='/';
				    	}
					
				    	int f=2*n-e+1;
				    	cout<<"|";
				      	for(int j=1;j<e;j++)
				    	{
				    		cout<<" ";
				    	}
				    	cout<<dh;
				    	for(int j=e+1;j<f;j++)
				    	{
				    		cout<<ch;
				    	}
				    	cout<<lh;
				    	for(int j=f+1;j<=2*n;j++)
				    	{
				    		cout<<" ";
				    	}
				    	cout<<"|";
				    }
			}
				
				
			}
		}
};
int main()
{
	pattern1 p;
	p.input();
	p.pattern();
}


